src/JsonApiDotNetCore/Builders/ContextGraphBuilder.cs

```diff
@@ -139,7 +141,7 @@ protected virtual List<AttrAttribute> GetAttributes(Type entityType)
 
             foreach (var prop in properties)
             {
-                if (prop.Name == nameof(Identifiable.Id))
+                if (prop.Name == nameof(Identifiable<int>.Id))
```

Comment:
The default non generic version of Identifiable has gone because of the special case code littered in JADNC v3 to support it. In this it doesn't matter since we're only using nameof to get the name of the Id property. In the N2 codebase I've already handled it where we used it previously in the Phase 2 PR